core/src/main/res/values-de/strings.xml

```diff
@@ -425,6 +425,7 @@
   <string name="pref_automatic_download_on_battery_sum">Automatische Downloads auch erlauben, wenn die Batterie nicht geladen wird</string>
   <string name="pref_parallel_downloads_title">Parallele Downloads</string>
   <string name="pref_episode_cache_title">Episodenspeicher</string>
+  <string name="pref_episode_cache_summary">Gesamtzahl aller heruntergeladenen Episoden die auf dem Gerät zwischengespeichert werden. Falls diese Zahl erreicht ist, werden automatische Downloads ausgesetzt.</string>
```

Comment:
Please remove the translation. Translations are only done using Transifex. Merging changes of sources and Transifex is cumbersome.

Reply:
Thank you for the comment. I removed the German translation.